### PR TITLE
Translate Google Summer of Code 2015 news (zh_cn) 

### PR DESCRIPTION
**Pull request by 1ternal**
translate google-summer-of-code-2015 news published at 2015-03-06

Related #1074

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho

Please review this pull request! :balloon:

File: zh_cn/news/_posts/2015-03-06-google-summer-of-code-2015.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: news_post
+title: "Google Summer of Code 2015"
+author: "Federico Builes"
+translator:
```
Comment (hlcfan) [inline zh_cn/news/_posts/2015-03-06-google-summer-of-code-2015.md]: Put your name here.

**Comment by 1ternal** [comment]
@hlcfan I've fix the issue, please review again.

**Comment by JuanitoFatas** [comment]
Looks except minor things, I fixed and merged as c245d73, thank you! And thanks @hlcfan for review.

**Comment by 1ternal** [comment]
@hlcfan , @JuanitoFatas thank you guys ! Actually, this is my first ever PR. :beers:  :smile:

**Comment by marocchino** [comment]
:clap: :clap:

**Comment by JuanitoFatas** [comment]
Awesome! :confetti_ball:

**Comment by JuanitoFatas** [comment]
@1ternal I found the title not translated, could you fix it and send another Pull Request? Thank you!

**Pull request closed without merging**